### DISCUSSION TITLE
q-select @new-value not triggering on mobile after lost focus

### DISCUSSION THREAD

**Discussion by michelsondan** [quasarframework/quasar#14720] [General - Components / Directives / etc] [answered]
Hello,
On kesktop if user press the enter key, @new-value is triggering,
but, On mobile, there is no enter key, and the user is moving to the next input, what should I do to get @new-value trigger.

Thank you.

**Comment by pdanpdan** [comment]
On mobile you also have the enter key

**Reply by michelsondan** [reply]
But, many users move to the next input, and they don't understand, why the value they just entered, disappears.

**Comment by pdanpdan** [comment]
Check:
https://github.com/quasarframework/quasar/discussions/13161
https://github.com/quasarframework/quasar/issues/5768
variations on https://quasar.dev/vue-components/select#example--selecting-option-after-filtering

**Reply by michelsondan** [reply]
I have tried on @blur to use setOptionIndex and moveOptionSelection and it is not working even for exsisting option.
Why can't we have a option to choose if on lostfocus, the value entered will be added as new value?

**Comment by pdanpdan** [comment, marked as answer]
https://codepen.io/pdanpdan/pen/gOKbjdE?editors=1010

**Reply by michelsondan** [reply]
Thank you. This solution solved my problem.
